content/c/concepts/preprocessors/preprocessors.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'Preprocessors'
+Description: 'A preprocessor scans the C program source code before it is compiled, allowing the source code to expand macros, include other files, and conditionally compile blocks of source code.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Command Line'
+  - 'Developer Tools'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+A preprocessor scans the C program source code before it is compiled, allowing the source code to expand macros, include other files, and conditionally compile blocks of source code.
```

Comment:
@yangc95 To [your point](https://github.com/Codecademy/docs/pull/490/files#r777049499), maybe we could like C Compiling to "compiled" in this section?